CONTRIBUTING.md

```diff
@@ -29,7 +29,7 @@ We use labels to track, manage and organize issues and pull requests.
 | `needs review`     | [search][needs review]  | Work still needs to be reviewed by another member of the team. |
 | `#pairwithme`      | [search][#pairwithme]   | Looking for someone to pair on this. |
 | `question`         | [search][question]      | Feature and code related questions which are no bug reports or feature requests. |
-| `rgsoc-<year>`     | [search][rgsoc-<year>]  | Issues targeting a specific `<year>` of the program, being relevant to e.g. the beginning of the program in 2016. |
+| `rgsoc-<year>`     | [search][rgsoc-<year>]  | Issues that a pair of RGSoC students of a given `<year>` working on the Teams App as their project can work on. |
```

Comment:
@klappradla come to think of it, those issues aren't really _reserved_. There are way more issues tagged with such a label than a pair of students could complete over the course of a summer. What about "Issues _intended_ for students working on the Teams App during RGSoC". Still has a lot of the "reserved"-semantics in it, though 🤔